### ISSUE TITLE
StructuredOutputParser does not contain min and max items from zodToJsonSchema 

### ISSUE THREAD

**Issue by Vaibhav-npc** [langchain-ai/langchainjs#4021]
.length property mentioned in zod schema does not get added to format_instructions that we send to LLM

**Comment by jacoblee93** [comment]
Thanks for the heads up!

@StefanTerdell I think unpinning may fix this as mentioned in #3983?

**Comment by StefanTerdell** [comment]
@jacoblee93 Yup! .length support was added after the current pinned version. I'm just about to look into unpinning it
